Snippets/Snippets_5/Outbox/SqlServer/ConnectionStrings.xml

```diff
@@ -1,6 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <!--startcode OutboxSqlServerConnectionStrings-->
 <configuration>
+  <configSections>
+    <section name="AuditConfig" type="NServiceBus.Config.AuditConfig, NServiceBus.Core"/>
```

Comment:
Not sure what do you mean? That this config section should also be mentioned in the sample description?

When we say how to verify the sample works, we mention to check if there are messages in the audit queue. There weren't, because this section was missing.